### DISCUSSION TITLE
r | w

### DISCUSSION THREAD

**Discussion by Mtigee** [Ma7moudTiger/Grow#26] [Q&A] [answered]
![image](https://github.com/Ma7moudTiger/Grow/assets/156967253/ef323be8-39bb-43e2-8497-e24e2cfc64b1)

**Comment by mtiger114** [comment, marked as answer]
![image](https://github.com/Ma7moudTiger/Grow/assets/105823344/5fbdd3a3-236c-4848-8141-4dc50cf6abf8)
instead
